manifests/init.pp

```diff
@@ -7,7 +7,8 @@
 # @author Trevor Vaughan <tvaughan@onyxpoint.com>
 #
 class postfix (
-  Boolean $enable_server = false
+  Boolean $enable_server = false,
+  Simplib::PackageEnsure $ensure = simplib::lookup('simp_options::package_ensure', { 'default_value' => 'installed' }),
```

Comment:
Ensure can pertain to many puppet resources, could we call this package_ensure, which also corresponds with the catalyst?